addons/note/static/src/xml/systray.xml

```diff
@@ -15,7 +15,7 @@
                         <div class="o_note_datetime"/>
                         <span class="ml4">
                             <a class="o_note_set_datetime text-muted">
-                                <span class="fa fa-clock-o" role="img" aria-label="Set date and time" title="Set date and time"/>
+                                <span class="fa fa-calendar" role="img" aria-label="Set date and time" title="Set date and time"/>
```

Comment:
`fa` icon is not visible because of `text-muted` (bootstrap 4):

```
.bg-dark .text-muted {
   color: rgba(255, 255, 255, 0.4) !important;
}
```

So the work-around is to remove the class `text-muted`, and define a style on `o_note_set_datetime`:

In note.scss:

```
.o_note.o_mail_preview {
   ...
   .o_note_datetime {
      .o_note_set_datetime {
         color: $text-muted;
      }
      ...
   }
}
```